validator/src/main/java/com/amazon/aoc/fileconfigs/PredefinedExpectedTemplate.java

```diff
@@ -194,6 +194,27 @@ public enum PredefinedExpectedTemplate implements FileConfig {
   DOTNET_EC2_DEFAULT_CLIENT_CALL_METRIC("/expected-data-template/dotnet/ec2/default/client-call-metric.mustache"),
   DOTNET_EC2_DEFAULT_CLIENT_CALL_TRACE("/expected-data-template/dotnet/ec2/default/client-call-trace.mustache"),
 
+  /** DotNet EC2 Windows Default Test Case Validations */
+  DOTNET_EC2_WINDOWS_DEFAULT_OUTGOING_HTTP_CALL_LOG(
+          "/expected-data-template/dotnet/ec2/windows/outgoing-http-call-log.mustache"),
+  DOTNET_EC2_WINDOWS_DEFAULT_OUTGOING_HTTP_CALL_METRIC(
+          "/expected-data-template/dotnet/ec2/windows/outgoing-http-call-metric.mustache"),
+  DOTNET_EC2_WINDOWS_DEFAULT_OUTGOING_HTTP_CALL_TRACE(
+          "/expected-data-template/dotnet/ec2/windows/outgoing-http-call-trace.mustache"),
+
+  DOTNET_EC2_WINDOWS_DEFAULT_AWS_SDK_CALL_LOG("/expected-data-template/dotnet/ec2/windows/aws-sdk-call-log.mustache"),
+  DOTNET_EC2_WINDOWS_DEFAULT_AWS_SDK_CALL_METRIC("/expected-data-template/dotnet/ec2/windows/aws-sdk-call-metric.mustache"),
+  DOTNET_EC2_WINDOWS_DEFAULT_AWS_SDK_CALL_TRACE("/expected-data-template/dotnet/ec2/windows/aws-sdk-call-trace.mustache"),
+
+  DOTNET_EC2_WINDOWS_DEFAULT_REMOTE_SERVICE_LOG("/expected-data-template/dotnet/ec2/windows/remote-service-log.mustache"),
+  DOTNET_EC2_WINDOWS_DEFAULT_REMOTE_SERVICE_METRIC("/expected-data-template/dotnet/ec2/windows/remote-service-metric.mustache"),
+//  Because of a time sync issue, block the remote service trace check for now
+//  DOTNET_EC2_WINDOWS_DEFAULT_REMOTE_SERVICE_TRACE("/expected-data-template/dotnet/ec2/windows/remote-service-trace.mustache"),
```

Comment:
What's the issue here?

Reply:
No sure root cause, there is a chance (1/4 chance or something, can be reduce to ~ 10% by put a delay but still a risk) for a same group of span, remote service (callee) have a timestamp earlier than main service (caller), not sure why,  maybe windows time sync related issue. Need to investigate a more accurate time synchronization method. Agreed to treat as a backlog item around weeks ago

Reply:
How does that impact the incoming traces?

Reply:
the trace check have subsegment [0] and [1]
[0] is the "first" segment by timestamp, namely and natively should be the caller
[1] is the "second" by timestamp, should be the remote (callee)
and unfortunately, there is a chance for the callee have earlier time-stamp, cause a reverse [0] and [1], cause a fail